unit_tests/library/tile.cpp

```diff
@@ -205,7 +205,7 @@ BOOST_AUTO_TEST_CASE(test_tile)
     }
 
     BOOST_CHECK_EQUAL(number_of_turn_keys, 3);
-    BOOST_CHECK_EQUAL(number_of_turns_found, 732);
+    BOOST_CHECK_EQUAL(number_of_turns_found, 763);
```

Comment:
Should we remove exact checks here - every other pull request has to adapt these numbers